src/views/List.jsx

```diff
@@ -1,14 +1,52 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	const onSearch = (e) => {
+		setSearchTerm(e.target.value);
+	};
+
+	const onSearchReset = () => {
+		setSearchTerm('');
+	};
+
+	const filteredData = data.filter((item) =>
+		item.name.toLowerCase().includes(searchTerm.toLowerCase()),
+	);
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+	};
+
 	// TODO - Use navigation similar to in Home.jsx to navigate to home page if no token or token is null
 	return (
 		<>
-			<ul>
-				{data.map((item, i) => (
-					<ListItem key={item.name + i} {...item} />
-				))}
-			</ul>
+			<form onSubmit={handleSubmit}>
+				<label htmlFor="searchTerm">Filter list items</label>
+				<input
+					type="text"
+					value={searchTerm}
+					id="searchTerm"
+					placeholder="Start typing here..."
+					onChange={onSearch}
+				></input>
```

Comment:
The smallest of critiques... but you could close the input on line 34 with just the  />
I think this would make it look just a little cleaner right here :)

Reply:
Oh, great call @mariahschock. I went ahead and made that change. All 'nits' welcome!